Description: Relax dependencies using Debian versions
Author: Arnaud Ferraris <aferraris@debian.org>
Last-Update: 2024-03-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -107,7 +107,7 @@
 optional = true
 
 [dev-dependencies.env_logger]
-version = "0.10.1"
+version = "0.10"
 
 [dev-dependencies.tokio]
 version = "1.35"
